PublicGitArchive/doc/dataset_analysis.md

```diff
@@ -0,0 +1,71 @@
+# Description of the current dataset
+
+## 1. GHTorrent: selecting the list of URLs
+
+From GHTorrent’s MySQL dump, dated from 01/01/2018.
+By querying repos with at least 50 stargazers
+
+Output: list of 190,051 URLs with duplicates  
+List of unique URLs: 187,352
+
+## 2. Collection with Borges
+
+Output of Borges:  
+
+183,015 **repositories**  
+4,338 are **indefinitely pending** for several reasons :  
+* private repos  
+* not existing anymore  
+* borges is not able to process them, maybe a bug
```

Comment:
@erizocosmico Are there really some repositories left which we are unable to clone?